tests/test/scheduler/test_mpi_world.cpp

```diff
@@ -1051,16 +1058,13 @@ template void doReduceTest<double>(scheduler::MpiWorld& world,
 
 TEST_CASE("Test reduce", "[mpi]")
 {
+    cleanFaabric();
```

Comment:
Would be nice to add a fixture to this file as we also have a custom `tearDown` method. One for another PR probably.